assets/app/views/_overview-deployment.html

```diff
@@ -42,11 +42,11 @@ <h3 class="h4">
       </h3>
     </div>
 
-    <div class="component meta-data">
+    <div class="component meta-data" ng-init="causes = (rc | deploymentCauses)">
```

Comment:
I actually meant to use the filter in the controller to pull data out of the deploymentConfig and continue setting it as a field on the rc, so the view can just do `rc.causes...`